qemu/tests/qemu_guest_agent.py

```diff
@@ -885,6 +885,28 @@ def gagent_check_fsfreeze(self, test, params, env):
         # Finally, do something after thaw.
         self._action_after_fsthaw(test, params, env)
 
+    @error_context.context_aware
+    def gagent_check_thaw_unfrozen(self, test, params, env):
+        """
+        Thaw the unfrozen fs
+
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environment.
+        """
+        try:
+            error_context.context("Verify if FS is thawed", logging.info)
+            expect_status = self.gagent.FSFREEZE_STATUS_THAWED
+            self.gagent.verify_fsfreeze_status(expect_status)
+        except guest_agent.VAgentFreezeStatusError:
+            # Thaw guest FS if the fs status isn't thawed.
+            self.gagent.fsthaw(check_status=False)
```

Comment:
`check_status=True` to verify the process.

Reply:
Yes, you are right. Updated.